package.json

```diff
@@ -42,18 +40,16 @@
     "bs58": "^4.0.1",
     "elliptic": "^6.5.4",
     "request": "^2.85.0",
-    "request-promise": "^4.2.6",
-    "underscore": "^1.13.2"
+    "request-promise-native": "^1.0.0"
```

Comment:
this package is deprecated

Reply:
Yes, all request packages are deprecated, including request itself, are deprecated. request-promise is also deprecated.

Reply:
Here's more info on it from the request team: https://github.com/request/request/issues/3142